### ISSUE TITLE
[Feature Request] Switch to Kotlin/material you/Compose

### ISSUE THREAD

**Issue by trlef19** [moneytoo/Player#610]
Kotlin is now the default android lang, same for MY & compose :)) Its okay if you dont want to though

**Comment by moneytoo** [comment]
I hear you. 

I wanted to and tried MD but I was unable to make use of it. The thing is that Just Player is basicaly single Activity. And that Activity is the player interface that ExoPlayer provides. I started with using accent color, tried applying it to the progress bar. And in some cases it was ok, but in the end the least neutral color (shades of gray) felt as the most suitable. Just Player dosn't really has a brand color to show. It's about the video content and not disturbing the user.
So it's basically only about the dialog windows. They are semi transparent now. I already make their corners rounded. MD would probably rounded it more but again, what suits the UI most is the more compact look.

Kotlin literally made me feel physical pain. I tried rewriting some parts to Kotlin in the past but the build times were significantly longer which heated my laptop (and my lap).

Compose. There's not much to do when basically the only thing is the reused interface from ExoPlayer which doesn't yet directly support Compose.
